spec/features/landing_face_to_face_filter_for_pension_advice_spec.rb

```diff
@@ -1,11 +1,11 @@
-RSpec.feature 'Consumer requires help with their pension in person' do
+RSpec.feature 'Lading page, consumer requires help with their pension in person' do
   let(:landing_page) { LandingPage.new }
   let(:results_page) { ResultsPage.new }
 
   let(:latitude) { rand(51.428473..55.856191) }
   let(:longitude) { rand(-4.247082..-0.943616) }
 
-  scenario 'Filter for general pension pot advice' do
+  scenario 'Filter for general pension pot advice returns matching firms' do
```

Comment:
I think the `returns matching firms` in all of these scenarios is already implied. Can we remove that bit of text here and elsewhere?